### DISCUSSION TITLE
Different rules for tests

### DISCUSSION THREAD

**Discussion by B4nan** [oxc-project/oxc#4663] [Q&A]
I'd like to lift the rules a bit for tests, e.g. I don't need to care about type imports there, while I do want to have that rule enabled for the actual code. With eslint, I just had a nested config in the tests folder, is there an alternative in oxlint? Or do I need to have two separate `oxlint` calls with different configs?

**Comment by Boshen** [comment]
This is finally on the roadmap. https://github.com/oxc-project/oxc/issues/5653
